migrations/versions/3905cabd818e_json_fields.py

```diff
@@ -0,0 +1,30 @@
+"""json fields
+
+Revision ID: 3905cabd818e
+Revises: 4b86d00d91d7
+Create Date: 2016-12-22 00:42:50.274064
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '3905cabd818e'
+down_revision = '4b86d00d91d7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('organizations', 'settings', type_=postgresql.JSONB, postgresql_using='settings::jsonb')
```

Comment:
JSONB was introduced in Postgres 9.4, while many Redash deployments are on Postgres 9.3.